### PR TITLE
Add tskit 0.4.1

### PR DESCRIPTION
**Pull request by benjeffery**
### Description

Add `tskit` 0.4.1 package and it's two pure-python deps.
No patches to tskit were needed.

### Checklists
- [x] Add a [CHANGELOG](https://github.com/pyodide/pyodide/blob/main/docs/project/changelog.md) entry
- [x] Add / update tests
- [x] ~Add new / update outdated documentation~

### CODE REVIEW

**Comment by hoodmane** [comment]
Can you merge the latest main? I just merged a fix for test-docs.

**Comment by hoodmane** [comment]
Yeah you should do `python -m pyodide_build mkpkg svgwrite` and add that.

File: packages/jsonschema/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+package:
+  name: jsonschema
+  version: 4.4.0
+source:
+  url: https://files.pythonhosted.org/packages/55/b2/2c4af6a97c3f12c6d5a72b41d328c3996e14e1e46701df3fac1ed65119c9/jsonschema-4.4.0-py3-none-any.whl
+  sha256: 77281a1f71684953ee8b3d488371b162419767973789272434bbc3f29d9c8823
```
Comment (ryanking13) [inline packages/jsonschema/meta.yaml:6]:
```suggestion
  sha256: 77281a1f71684953ee8b3d488371b162419767973789272434bbc3f29d9c8823
requirments:
  run:
    - attrs
```

**Review by ryanking13 — COMMENTED** [review]
It looks like `jsonschema` requires `attrs` to run.

File: cpython/remove_modules.txt [hunk 2]
```diff
@@ -1,12 +0,0 @@
-_osx_support.py
-curses
```
Comment (ryanking13) [inline cpython/remove_modules.txt:2]: Could you revert this change if you removed this file by a mistake?

Comment (benjeffery) [reply cpython/remove_modules.txt]: Whoops, was from when I was trying to get around #2449. Fixed.

**Review by ryanking13 — APPROVED** [review]
The remaining error is from the tskit test. Please update the test. Except that, LGTM. Thanks @benjeffery!

**Comment by ryanking13** [comment]
Thanks @benjeffery!

**Comment by benjeffery** [comment]
> Thanks @benjeffery!

Thank you! Docs and great CI made this easy.

File: packages/tskit/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,22 @@
+package:
+  name: tskit
+  version: 0.4.1
+source:
+  url: https://files.pythonhosted.org/packages/fb/0d/00407181bf409da76dd9a63741d54d57cfa6afec42957ca6a6b93dbfb530/tskit-0.4.1.tar.gz
+  sha256: 28b07fb7f7b19dadd133ab49938c2a38e1558b1cb76122e57f4d1d1230c383ca
+build:
+  script: |
+    pip install -t $HOSTSITEPACKAGES numpy
```
Comment (rth) [inline packages/tskit/meta.yaml:9]: Actually, this should not be necessary, as discussed in https://github.com/pyodide/pyodide/pull/2548#discussion_r874083146 so we should revert this line.
